### PR TITLE
chore: archive v1.5.x

### PR DESCRIPTION
**Pull request by derekbit**
#### Which issue(s) this PR fixes:

Issue #

1.5.x is EOL

### CODE REVIEW

**Comment by derekbit** [comment]
> @derekbit PR checks failed.

Fixed and ready for review.

**Comment by derekbit** [comment]
ping @jhkrug

File: config.toml [hunk 1]
```diff
@@ -14,8 +14,8 @@ style = "paraiso-dark"
 [params]
 description = "Cloud native distributed block storage for Kubernetes"
 tagline = "Easy to use, 100% open source, run anywhere"
-versions = ["1.7.2", "1.7.1", "1.7.0", "1.6.3", "1.6.2", "1.6.1", "1.6.0", "1.5.5", "1.5.4", "1.5.3", "1.5.2", "1.5.1", "1.5.0", "1.8.0-dev", "1.7.3-dev", "1.6.4-dev", "1.5.6-dev"]
-archived_versions = ["1.4.4", "1.4.3", "1.4.2", "1.4.1", "1.4.0", "1.3.3", "1.3.2", "1.3.1", "1.3.0", "1.2.6", "1.1.3"]
+versions = ["1.7.2", "1.7.1", "1.7.0", "1.6.3", "1.6.2", "1.6.1", "1.6.0", "1.8.0-dev", "1.7.3-dev", "1.6.4-dev"]
+archived_versions = ["1.5.5", "1.5.4", "1.5.3", "1.5.2", "1.5.1", "1.5.0", "1.4.4", "1.4.3", "1.4.2", "1.4.1", "1.4.0", "1.3.3", "1.3.2", "1.3.1", "1.3.0", "1.2.6", "1.1.3"]
```
Comment (innobead) [inline config.toml:18]: For 1.4 and 1.3, let's just keep the last version, 1.4.4 and 1.3.3.

**Review by innobead — CHANGES_REQUESTED** [review]

**Review by innobead — APPROVED** [review]

**Comment by innobead** [comment]
This is a config change w/o doc updates. Merged w/o @longhorn/doc review.

**Pull request merged**